configs/build_sunspot.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+module load spack
+module load cmake
+module load oneapi/eng-compiler/2023.05.15.007
```

Comment:
there are multiple compatibility issue between compilers and libraries.
So please load the `frameworks/2023.05.15.001` module and let it decide compiler.
Also load cmake after loading the framework module.